communities/UNAM-Engineering/community-assessment.md

```diff
@@ -0,0 +1,162 @@
+  - Students interested in developing projects in their free time.
+  - Students already working on research and/or developing apps.
+  - Students interested to attend hackathons and meetups.
+  - Students interested in landing internship opportunities.
+  - Students interested in meeting industry companies and people.
+
+  
+### Size
+- How many people are currently in your community?
+
+Currently there's 45 people on the GitHub organization and around 10 people have submitted repos.
+
+- How many would you _like_ to be in your community?
+
+Getting to 150 people in the organization and having a 20% - 25% members contributing.
+
+- Historically, has the group always been about the same size? If not, what brought on that change?
+
+The group grew a lot in the beginning and people used to share they're code. After four months the growth
+stopped because we didn't promote it outside of our core friends. Usually, the last couple months of the semester
+we get a lot of traffic from students looking for final projects done on previous years.
+
+### Resources
+- What is currently available to people in your community to learn about tech?
+
+We have some guides and projects done in the organization which help students understand the topics they see
+in their classes as well as some small courses.
+
+- Are there meetups, seminars, online tools, office hours, hackathons?
+
+Currently, we don't promote meetups nor seminars nor hackathons but is something we look forward to do. We have online tools. 
+We do office hours but not in an organized structure. Most of the time students come to the lab and ask for help.
+
+### Accessibility
+- How accessible are the current resources in your community?
+
+All of the community resources are open sourced.
+
+- Are meetings held during hours where people may be working?
+
+We don't hold meetups so far.
+
+- Is there childcare available?
+
+We don't have childcare available.
+
+### Interests
+- Is your community made up only of computer science students? Are you OK with that
+
+As a community we're open to anyone who wants to join though a bast mayority are computer engineers.
+
+- Is your community open to other topics, or are their interests very specific
+
+We're open to projects that are related to CS or CE so we might talk about other topics and how people 
+solved them with code.
+
+### Goals
+- Why do people get involved in your community
+
+Most of the people that join is to share their code with the organization or to ask for code exmaples about
+their classes.
+
+- Are they students hoping to get a degree
+
+Yes, most of them. We also have graduates as part of the community.
+
+- Are they trying to ramp-up to get a job
+
+Most are looking for internships for summer/winter.
+
+- Are they passionate about developing, regardless of their major
+
+Yes, we have also psychologist and law students that love to code.
```

Comment:
This is great! Do you do anything to champion the contributions of folks from non-traditional coding degrees?